README.md

```diff
@@ -41,9 +44,19 @@ minification {
     cssSrcDir = "${rootDir}/src/main/resources/static/css"
     jsDstDir = "$buildDir/dist/js"
     jsSrcDir = "${rootDir}/src/main/resources/static/js"
+
+    createCssSourceMaps = true
+    createJsSourceMaps = true
 }
 ````
 
+## Options
+| option                      | effect                                                   | values                          | default         |
+| --------------------------- |:--------------------------------------------------------:| -------------------------------:| ---------------:|
+| createCssSourceMaps         | Creates CSS source maps                                  | true, false                     | false           |
+| createJsSourceMaps          | Creates JS source maps                                   | true, false                     | false           |
+| originalFileNames           | Preserves filenames instead of renaming to .min.         | true, false                     | true            |
```

Comment:
I guess default should be false because in most use cases it makes more sense to rename files to .min.*

Reply:
That was my plan but I somehow switched it accidentally.
Fixed now